communication.md

```diff
@@ -51,19 +51,23 @@ Map that to your local time with this [timezone table].
 
 ## Conferences
 
-Kubernetes is the main focus of CloudNativeCon/KubeCon, held every spring in Europe and winter in North America. Information about these and other community events is available on the CNCF [events] pages.
+Helm is one of the projects, represented at CloudNativeCon/KubeCon, held three times per year in Europe, Asia and North America. Information about these and other community events is available on the CNCF [events] pages.
 
+Also, Helm community origanizes the Helm-focused event - [Helm Summit], first
```

Comment:
"Helm community" -> "the Helm community"
"origanizes" -> "organizes"

Reply:
@prydonius updated